addons/mail/static/src/models/composer/composer.js

```diff
@@ -19,18 +19,66 @@ function factory(dependencies) {
         // Public
         //----------------------------------------------------------------------
 
-        closeMentionSuggestions() {
+        closeSuggestions() {
```

Comment:
If we use compute for everything as suggested, this whole method may become just writing `delimiter` to empty.